src/entities/Proposal/types.ts

```diff
@@ -464,6 +464,7 @@ export enum ProposalGrantTier {
   Tier5 = 'Tier 5: up to $120,000 USD, 6 months vesting (1 month cliff)',
   Tier6 = 'Tier 6: up to $240,000 USD, 6 months vesting (1 month cliff)',
 }
+export const ONE_TIME_PAYMENT_TIERS = new Set([ProposalGrantTier.Tier1, ProposalGrantTier.Tier2])
```

Comment:
Maybe this can be in `Proposal/utills.ts`

Reply:
Yeah, initially I put it there, but I thought it might be better to have it next to the tier and sizes declarations...